client/src/components/Results/ResultsMap.js

```diff
@@ -22,7 +20,56 @@ const styles = {
   },
 };
 
-const useStyles = makeStyles((theme, props) => ({
+// the map layer that shows the bubbles
+const clusterLayer = {
+  id: "clusters",
+  type: "circle",
+  source: "stakeholders",
+  filter: ["has", "point_count"],
+  paint: {
+    "circle-color": [
+      "step",
+      ["get", "point_count"],
+      "#51bbd6",
+      100,
+      "#f1f075",
+      750,
+      "#f28cb1",
+    ],
+    "circle-radius": ["step", ["get", "point_count"], 20, 100, 30, 750, 40],
+  },
+};
+
+// the map layer that shows the numbers on the cluster bubbles
+const clusterCountLayer = {
+  id: "cluster-count",
+  type: "symbol",
+  source: "stakeholders",
+  filter: ["has", "point_count"],
+  layout: {
+    "text-field": "{point_count_abbreviated}",
+    "text-font": ["DIN Offc Pro Medium", "Arial Unicode MS Bold"],
```

Comment:
Let's also refer to the main theme for fonts. Feel free to update the theme file as needed